.env.test

```diff
@@ -10,6 +10,10 @@ DB_HOST=db
 REDIS_HOST=redis
 REDIS_PORT=6379
 REDIS_DB=0
+STATSD_BACKEND=markus.backends.datadog.DatadogMetrics
+STATSD_HOST=statsd
```

Comment:
Is specifying the actual connection params necessary for the tests to pass?  If it's not I'd rather omit it here so it's obvious that the tests don't need to connect to an actual statsd host, and move these to .env.sample

Reply:
I'll remove it... it's not necessary. If it complains I can just use the logging backend.